src/librustdoc/html/templates/item_union.html

```diff
@@ -10,7 +10,7 @@ <h2 id="fields" class="fields section-header"> {# #}
     {% for (field, ty) in self.fields_iter() %}
         {% let name = field.name.expect("union field name") %}
         <span id="structfield.{{ name }}" {#+ #}
-            class="{{ ItemType::StructField +}} section-header"> {# #}
+            class="structfield {{ ItemType::StructField +}} section-header"> {# #}
```

Comment:
I believe the `structfield` class is already applied using `ItemType::StructField`.

```suggestion
            class="{{ ItemType::StructField +}} section-header"> {# #}
```